app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/TaxYear.scala

```diff
@@ -30,6 +30,7 @@ final case class TaxYear(
   annualExemptAmountNonVulnerableTrust: AmountInPence,
   personalAllowance: AmountInPence,
   maxPersonalAllowance: AmountInPence,
+  higherIncomePersonalAllowanceThreshold: AmountInPence,
```

Comment:
does the BE not need updating to pass this value onto the FE?

Reply:
was already Done, just forgot to PR.